content/tutorials/webrtc/basics/en/index.html

```diff
@@ -27,125 +27,127 @@
 
 <h2 id="toc-disruptive">Real-time communication without plugins</h2>
 
-<p>Imagine a world where your phone, TV and computer could all communicate on a common platform. Imagine it was easy to add video chat and peer-to-peer data sharing to your web application. That's the vision of WebRTC.</p>
+<p>Imagine a world where your phone, TV, and computer could communicate on a common platform. Imagine it was easy to add video chat and peer-to-peer data sharing to your web app. That's the vision of WebRTC.</p>
 
-<p>Want to try it out? WebRTC is available now in Google Chrome, Safari, Firefox and Opera, on desktop and mobile. A good place to start is the simple video chat application at <a href="https://appr.tc" title="Simple WebRTC demo" target="_blank">appr.tc</a>:</p>
+<p>Want to try it out? WebRTC is available on desktop and mobile in Google Chrome, Safari, Firefox, and Opera. A good place to start is the simple video chat app at <a href="https://appr.tc" title="Simple WebRTC demo" target="_blank">appr.tc</a>:</p>
 
 <ol>
   <li>Open <a href="https://appr.tc" title="Simple WebRTC demo" target="_blank">appr.tc</a> in your browser.</li>
-  <li>Click the Join button to join a chat room and let the app use your webcam.</li>
-  <li>Open the URL displayed at the bottom of the page in a new tab or, better still, on a different computer.</li>
+  <li>Click <b>Join</b> to join a chat room and let the app use your webcam.</li>
```

Comment:
Nit: this will work fine, but best to use the semantic element `<strong>`.

Reply:
The style guide for developer documentation recommends <b> for bold formatting, so I think I'm going to leave this one as is.